Explicitly list settings that are not synced (#1796)
GoogleChrome/developer.chrome.com: site/en/docs/devtools/customize/index.md

    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yhIipqtEvDuy6ygB677t.png", alt="Chrome profile sync", width="400", height="579" %}
2. The DevTools sync settings can be updated via **Settings** > **Sync** > **Enable settings sync** checkbox.
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LUwFNTDyP22L1euSGg73.png", alt="DevTools sync settings", width="800", height="654" %}

> 3. Most of the settings you see in the **Settings** UI are synced, except for the **Workspace**, **Experiments**, **Devices** tab and a few other general settings. The state of the **Enable settings sync** checkbox is synced across devices as well.

Reviewer: ```suggestion
3. Most of the settings are synced, except for the **Workspace**, **Experiments**, and **Devices** tabs and a few other general settings. The state of the **Enable settings sync** checkbox is synced across devices as well.
```

Reviewer: >you see in the **Settings** UI 

We know this from the beginning of the sentence and from the context.

>   For example, the following **appearance** settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again.

Reviewer: ```suggestion
Additionally, the following **appearance** settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again.
```

Reviewer 2: This line is an explanation of the previous line. We mentioned some settings are sync, some aren't. Here are the examples of sync/not sync.
